resources/public/index.html

```diff
@@ -16,10 +16,7 @@
         <script src="cljs-out/dev/main_bundle.js" type="text/javascript"></script>
         <script type="text/javascript">
          //<![CDATA[
-         com.yetanalytics.lrs_admin_ui.core.init(
-             "serverHost", "http://localhost:8080",
-             "xapiPrefix", "/xapi"
-         );
+         com.yetanalytics.lrs_admin_ui.core.init("http://localhost:8080");
```

Comment:
out of curiosity, how hard would it be to use a relative path w/no arg? Would simplify the build a little when it just gets used on-lrs

Reply:
to be clear, I know it's not the point of this PR just curious

Reply:
it does use a relative path with no arg in the majority case, see prod_index. This is for weird dev scenarios where you gotta override because you're running the server in figwheel.

Reply:
gotcha